content/04.model-complexity.md

```diff
@@ -31,4 +31,24 @@ In immune cell signature discovery, elastic-net regression has been used to redu
 A variation of elastic-net, where a two-step regularized logistic regression was used to pre-select an optimal number of genes before implementing elastic-net regularization for gene selection, identified immune cell signatures in an RNA-seq dataset where the number of cells sampled were far fewer than number of genes profiled [@doi: 10.1186/s12859-019-2994-z].  
 Thus, regularization methods like LASSO or elastic-net have been methods of choice where the profiled feature space is substantially larger than the number of samples; these methods should be explored while working with rare disease datasets.
 
-#### Hyperensemble
+#### Ensemble learning
+
+Rare disease datasets not only present few data points but the data points also come with high label-uncertainty thus forming silver standard datasets.
```

Comment:
It _very well_ may have been my suggestion to kick the gold standard vs. silver standard discussion to the introduction but seeing this makes me think there's a non-zero chance that if someone is not familiar with this concept prior to reading the paper, they will have forgotten what it means by the time they make it to this section.

Reply:
```suggestion
Rare disease datasets not only present few data points but due to inadequate understanding of the diseases, the labels associated with the data points carry high uncertainty. 
Training ML models using data points with high label-uncertainty (i.e. a silver standard dataset) can lead to unstable predictions.
```

Reply:
rephrased this